libs/agent-sm/agent-policy/src/main/java/org/opensearch/secure_sm/policy/package-info.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
nit: shall the package name be `secure-sm` to be consistent?

Reply:
The hyphen (`-`) is not allowed in package names (and in general, in Java variable names as well): https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
i see, we have been skipping underscore as well in other package names like `ratelimitting`